src/pages/gift/about/FAQ.js

```diff
@@ -9,9 +9,9 @@ export default function FAQ() {
   const q6 = 'I am having trouble claiming my gift.';
   const a1 = (
     <p className="pb-3">
-      Polkadot is a blockchain, that connects other blockchains. DOT is the
```

Comment:
The exact language will probably get finalised by some of our writers. However, if our audience are users totally new to the ecosystem, the language used should probably be rather simple than overly technical

Reply:
To me, there will be two groups of users that are going to use this app. The people that are sending the gifts who are more likely blockchain savvy and have used polkadot before and on the other side the gift recipients who are new to the network and probably have not used polkadot or any other blockchains at all. The content creation team can hopefully help to keep the balance in targeting the both group of audiences. 
I also have this item #8  open to add some more info regarding account creation that should be targeted for the users that are new to the ecosystem.

Reply:
Yes, I agree with the two different groups of users, and need to balance language!